lib/job.js

```diff
@@ -275,7 +275,8 @@ class Job extends BaseModel {
         }
 
         options.paginate.page = 1;
-        const allBuilds = await getAllRecords.apply(this, ['getBuilds', options, [[]]]);
+        const allBuilds = await getAllRecords.apply(this,
+            ['getBuilds', config.aggregate, options, [[]]]);
```

Comment:
wondering if u know all the args, why not use `.call`?

Reply:
I guess I can use either. Are there any difference performance difference?

Reply:
Technical answer: yes
Is it a lot? Depends on number of args
Is this blocking? No, hence this is just a comment ;-P

Reply:
Just curious, which one has better performance?

Reply:
Call is bit faster than apply